README.md

```diff
@@ -1 +1,99 @@
-Coming soon…
+# UI Component Best Practices <!-- omit in toc -->
+
+A bunch of best practices to help achieve User Interface (UI) components, or
+presentational components, that are:
+
+- accessible
+- consumable
+- portable
+- reusable
+- and robust.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Caveats](#caveats)
+- [The tl;dr Version](#the-tldr-version)
+- [Components Cannot Apply External Spacing](#components-cannot-apply-external-spacing)
+  - [The Rule](#the-rule)
+  - [Why?](#why)
+    - [Why `margin` Specifically?](#why-margin-specifically)
+  - [The Exception(s)](#the-exceptions)
+  - [More Reading](#more-reading)
+
+## Caveats
+
+1. These best practices target components that are part of an organisation's component library or broader design system where the design is known.
+2. All code snippets are from the [React library](https://reactjs.org/).
+   However, this checklist is library/framework-agnostic. Meaning it can apply
+   to any component architecture, including [Web
+   Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
+
+## The tl;dr Version
+
+- [ ] Components cannot apply external spacing.
+
+## Components Cannot Apply External Spacing
+
+### The Rule
+
+The root element of a component must not use the `margin` property.
+
+### Why?
+
+Consider these components being rendered in a view layer:
+
+```jsx
+<Search />
+<SearchResults />
+<Pagination />
+<Downloads />
+```
+
+There needs to be `40px` of space between each component. We can easily achieve
+this by adding `margin-bottom: 40px;` to each of the component's root element.
+However, that would be short sighted as the
```

Comment:
Seems like your lines are breaking prematurely in a few spots

Reply:
Added Prettier.